### ISSUE TITLE
False positive in JJWT

### ISSUE THREAD

**Issue by stevespringett** [dependency-check/DependencyCheck#664]
JJWT, a Java and Android library for the generation of validation of JSON Web Tokens is being incorrectly identified as cpe:/a:sonatype:nexus due to the following two pieces of evidence:

```
Manifest
Bundle-Description
Sonatype helps open source projects to set up Maven repositories on https://oss.sonatype.org/
```
```
Manifest
implementation-url
http://nexus.sonatype.org/oss-repository-hosting.html/jjwt
```

**Comment by jeremylong** [comment]
@stevespringett This false positive may crop up in other libraries that utilize the [Sonatype OSS parent pom](http://search.maven.org/#artifactdetails%7Corg.sonatype.oss%7Coss-parent%7C9%7Cpom).  The way things are configured for JJWT is that some of the parent pom data is ending up in the manifest. 

Rather then simply adding a suppression rule for each one that we find it might be better to update the hint analyzer to remove evidence based on a regex value instead of the exact match mechanism used now. An example of the current way to remove single items of evidence can be found in the [dependencycheck-base-hint.xml](https://github.com/jeremylong/DependencyCheck/blob/master/dependency-check-core/src/main/resources/dependencycheck-base-hint.xml#L131).  I'll try to work on this shortly.

**Comment by jeremylong** [comment]
Closing this issue as the suppression rule has been added. The enhancement request has been added as #704.
